template/.env.example

```diff
@@ -34,11 +34,19 @@ AUTH_SALT=''
 SECURE_AUTH_SALT=''
 LOGGED_IN_SALT=''
 NONCE_SALT=''
-
 <%_ if (wpRocket) { _%>
 
 # WP Rocket config
 WP_ROCKET_EMAIL=''
 WP_ROCKET_KEY=''
 WP_CACHE=false
 <%_ } _%>
+<%_ if (sentry) { _%>
+
+# Sentry config (https://github.com/stayallive/wp-sentry)
+WP_SENTRY_PHP_DSN=''
+WP_SENTRY_BROWSER_DSN=''
+WP_SENTRY_BROWSER_TRACES_SAMPLE_RATE=false
+WP_SENTRY_ENV=''
+WP_SENTRY_VERSION=''
```

Comment:
**nitpick:** Remove `WP_SENTRY_VERSION` 

It is never used as the version is read from the `package.json` file